config/ext/rspec.yml

```diff
@@ -636,6 +645,12 @@ RSpec/MissingExampleGroupArgument:
   VersionAdded: '1.28'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/MissingExampleGroupArgument
 
+RSpec/MissingExpectationTargetMethod:
```

Comment:
https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecmissingexpectationtargetmethod :+1:

Reply:
I assume for `should`-syntax, this Cop is ignored, right? Because otherwise it would mark all cases where you still use `should`